float32 joyleft_x
float32 joyleft_y
float32 joyright_x
float32 joyright_y
bool up
bool down
bool left
bool right
bool triangle
bool x
bool square
bool circle
bool select
bool start
bool playstation
bool L1
float32 L2
bool R1
float32 R2
